src/map/mob.cpp

```diff
@@ -3667,7 +3668,7 @@ bool mob_chat_display_message(mob_data &md, uint16 msg_id) {
 /*==========================================
  * Skill use judging
  *------------------------------------------*/
-int mobskill_use(struct mob_data *md, t_tick tick, int event)
+int mobskill_use(struct mob_data *md, t_tick tick, int event, int64 damage)
```

Comment:
Why int64 ? `damage` from `mob_damage` is int

Reply:
I guess that was left from trying to use battle_damage :)